modules/virt-customizing-storage-profile.adoc

```diff
@@ -22,7 +22,7 @@ If you create a data volume and omit YAML attributes and these attributes are no
 
 .Procedure
 
-. Edit the storage profile. In this example, the provisioner is not recognized by CDI:
+. Edit the storage profile. In this example, the provisioner is not recognized by CDI.
 +
 [source,terminal,subs="attributes+"]
 ----
```

Comment:
`$ oc edit storageprofile <storage_class>`

Because storageprofile is not a namespaced resource